src/test/unit_test_runner.cpp

```diff
@@ -0,0 +1,2 @@
+#define CATCH_CONFIG_MAIN
```

Comment:
Can we add this to the CI?

Reply:
Yes, but let's do that in a different PR. see #457

Reply:
nm, I added it along with the comments/reversions/adjustments pushed. Thanks for your feedback!